taskcluster/scripts/toolchain/external-gradle-dependencies/after.sh

```diff
@@ -32,4 +33,18 @@ function _package_artifacts_downloaded_by_nexus() {
 }
 
 
+function _ensure_artifacts_are_sane() {
+    : > "$BAD_DOWNLOADS_FILE"
+    # Let's find empty files or unfinished downloads
+    find "$WORKSPACE/$EXTERNAL_DEPS" -size 0 -o -name '*.part' >> "$BAD_DOWNLOADS_FILE"
+
+    if [[ -s "$BAD_DOWNLOADS_FILE" ]]; then
```

Comment:
I'd use `[ -s ... ]` instead of `[[ ... ]]` unless there's a particular reason not to?